app/controllers/proxy_rule_shared_controller.rb

```diff
@@ -14,7 +14,7 @@ def self.included(base)
   def index
     @search = ThreeScale::Search.new(params[:search])
     query   = ProxyRuleQuery.new(owner_type: params[:owner_type], owner_id: owner_id,
-                                 direction: params[:direction], sort: params[:sort], per_page: per_page)
+                                 direction: params[:direction], sort: params[:sort])
```

Comment:
From what I could gather, this is the only place in code where we call `ProxyRuleQuery` directly. If we're to stop providing a `per_page` argument to it, maybe we should remove it from the class' `initialize` method as well.

Reply:
Makes sense :+1:

Reply:
I thought about this as well but `per_page` should still be a valid argument. If we remove it now, we'll have to add it back again when implementing Patternfly 4 in this page (where tables have a "per page" option).